Incomplete JAR installer for eXist-db 7.0.0-SNAPSHOT for Windows

**Incomplete JAR installer for eXist-db 7.0.0-SNAPSHOT for Windows**
JAR installer generated from the current `develop` branch installs eXist-db, but the database contains only `system` collection, and no `apps` collection.

**Expected behavior**
After installation, the installed eXist-db application should be complete and functional.

**To Reproduce**
- build eXist-db from the `develop` branch on Windows machine
  - configuration:  
    - Microsoft Windows 10 Pro, 10.0.19045 N/A Build 1904
    - PowerShell 7.6.2.
    - openjdk 21.0.10 2026-01-20 LTS
    - Apache Maven 3.9.16
 - command:
   - `$env:JAVA_HOME="C:\Program Files\Android\openjdk\jdk-21.0.8"; mvn -Prelease-build -Drevision="7.0.0-SNAPSHOT" -DskipTests "-Ddependency-check.skip=true" "-Dlicense.skip=true" clean package`


**Additional context**
- following log is generated
```log
[INFO] Following Maven properties got included in the installer:
apache.httpcomponents.core.version=4.4.16, apache.httpcomponents.version=4.5.14, apache.xmlrpc.version=5.0.0, appassembler.version=2.1.0, aspectj.version=1.9.25.1, contact.email=info@exist-db.org, exquery.distribution.version=0.2.1, git.closest.tag.commit.count=, git.closest.tag.name=, git.commit.id.abbrev=118bc1e, git.commit.id.describe=118bc1e, git.commit.id=118bc1e4227c73cd9dbd0cffb9af2c95870bd7ad, git.commit.time=20260602115926, icu.version=76.1, izpack.installation.info.appversion=7.0.0-SNAPSHOT, izpack.installation.info.author.email=info@exist-db.org, izpack.installation.info.author.name=The eXist-db Open Source project, izpack.installation.info.url=http://www.exist-db.org, izpack.resources.src=V:\Projekty\Github\eXist-db\exist\exist-installer/src/main/izpack, izpack.resources.target=V:\Projekty\Github\eXist-db\exist\exist-installer/target/izpack, izpack.version=5.2.5, jansi.version=4.1.3, jaxb.api.version=4.0.5, jaxb.impl.version=4.0.2, jetty.version=12.1.9, log4j.version=2.26.0, lucene.version=10.4.0, maven.compiler.release=21, milton.version=1.8.1.3, project.build.sourceEncoding=UTF-8, project.copyright.name=The eXist-db Authors, saxon.version=12.5, xmlresolver.version=6.0.23
[INFO] Pack compression method: bzip2
[INFO] Setting the installer information
[INFO] Adding uninstaller
[INFO] Setting the installer information
[INFO] Setting the console preferences
[INFO] Setting the GUI preferences
[INFO] Adding langpack: eng
[INFO] Adding resource: flag.eng
[INFO] Adding resource: logging.properties
[INFO] Adding resource: Installer.image
[INFO] Adding resource: shortcutSpec.xml
[INFO] Adding resource: Unix_shortcutSpec.xml
[INFO] Adding resource: HTMLInfoPanel.info
[INFO] Adding resource: userInputSpec.xml
[INFO] Adding resource: ProcessPanel.Spec.xml
[INFO] Adding resource: HTMLInfoPanel.start
[INFO] Adding resource: CustomLangPack.xml_eng
[INFO] Adding resource: logo
[INFO] Adding panel: start :: Classname : com.izforge.izpack.panels.htmlinfo.HTMLInfoPanel
[INFO] Adding panel: TargetPanel.0 :: Classname : com.izforge.izpack.panels.target.TargetPanel
[INFO] Adding panel: DataDirPanel :: Classname : com.izforge.izpack.panels.userinput.UserInputPanel
[INFO] Adding panel: AdminPasswordPanel :: Classname : com.izforge.izpack.panels.userinput.UserInputPanel
[INFO] Adding panel: TreePacksPanel.0 :: Classname : com.izforge.izpack.panels.treepacks.TreePacksPanel
[INFO] Adding panel: InstallPanel.0 :: Classname : com.izforge.izpack.panels.install.InstallPanel
[INFO] Adding panel: ProcessPanel.0 :: Classname : com.izforge.izpack.panels.process.ProcessPanel
[INFO] Adding panel: ShortcutPanel.0 :: Classname : com.izforge.izpack.panels.shortcut.ShortcutPanel
[INFO] Adding panel: info :: Classname : com.izforge.izpack.panels.htmlinfo.HTMLInfoPanel
[INFO] Adding pack temp containing 1 files
[INFO] Marked target file executable: bin/backup.sh
[INFO] Marked target file executable: bin/client.sh
[INFO] Marked target file executable: bin/export.sh
[INFO] Marked target file executable: bin/export-gui.sh
[INFO] Marked target file executable: bin/jmxclient.sh
[INFO] Marked target file executable: bin/launcher.sh
[INFO] Marked target file executable: bin/shutdown.sh
[INFO] Marked target file executable: bin/startup.sh
[INFO] Marked target file executable: bin/jsvc-linux-x86_64
[INFO] Marked target file executable: bin/jsvc-mac
[INFO] Adding pack Server containing 491 files
[INFO] Adding pack Apps containing 0 files
[INFO] Adding pack templating containing 1 files
[INFO] Adding pack Semver.xq containing 1 files
[INFO] Adding pack dashboard containing 1 files
[INFO] Adding pack eXide containing 1 files
[INFO] Adding pack functx containing 1 files
[INFO] Adding pack monex containing 1 files
[INFO] Adding pack packageservice containing 1 files
[INFO] Adding pack exist-documentation containing 1 files
[INFO] Adding pack exist-function-documentation containing 1 files
[INFO] [ Begin ]
[INFO]
[INFO] Copying the skeleton installer
[INFO] Copying 12 files into installer
[INFO] Writing 12 Packs into installer
[INFO] Writing Pack 0: temp
[INFO] Writing Pack 1: Server
[INFO] Writing Pack 2: Apps
[INFO] Writing Pack 3: templating
[INFO] Writing Pack 4: Semver.xq
[INFO] Writing Pack 5: dashboard
[INFO] Writing Pack 6: eXide
[INFO] Writing Pack 7: functx
[INFO] Writing Pack 8: monex
[INFO] Writing Pack 9: packageservice
[INFO] Writing Pack 10: exist-documentation
[INFO] Writing Pack 11: exist-function-documentation
[INFO]
[INFO] [ End ]
[INFO]
[INFO] --- source:3.4.0:jar-no-fork (attach-sources) @ exist-installer ---
[INFO] No sources in project. Archive not created.
[INFO]
[INFO] --- javadoc:3.12.0:jar (attach-javadocs) @ exist-installer ---
[INFO] No Javadoc in project. Archive not created.
```
- comparison of generated distribution (`exist-distribution-7.0.0-SNAPSHOT-win.zip`) and installer (`exist-installer-7.0.0-SNAPSHOT.jar`) shows that distribution `autodeploy` directory contains more packages than `packs` directory of the installer
  -  existdb-openapi-0.9.7.xar
  - roaster-1.12.1.xar
- In the distribution, the package is named `semver-xq-3.0.0.xar`, but in the installer it is called `pack-Semver.xq` (instead of `pack-semver-xq` ?)
- running eXist-db from **generated distribution** (`exist-distribution-7.0.0-SNAPSHOT-win.zip`) also creates eXist-db instance without `apps` collection

This time (`1a1091e3`) it fails with

```log
[INFO] --- copy-rename:1.0:rename (rename-jetty-etc-dir-for-appassembler) @ exist-distribution ---
[ERROR] sourceFile V:\Projekty\Github\eXist-db\exist\exist-distribution\target\exist-distribution-dir\etc\org\exist\jetty\etc does not exist
....
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal org.codehaus.mojo:xml-maven-plugin:1.2.1:transform (adjust-jetty-deploy-for-appassembler) on project exist-distribution: The directory V:\Projekty\Github\eXist-db\exist\exist-distribution\target\exist-distribution-dir\etc\jetty, which is a base directory of a ValidationSet or TransformationSet, does not exist. -> [Help 1]
```

Do you need more log, @duncdrum ?

--- a/exist-installer/src/main/izpack/install.xml
+++ b/exist-installer/src/main/izpack/install.xml
@@ -41,7 +41,7 @@
         <javaversion>@{maven.compiler.release}</javaversion>
         <requiresjdk>no</requiresjdk>
         <run-privileged condition="izpack.windowsinstall"/>
-        <pack-compression-format>bzip2</pack-compression-format>
+        <pack-compression-format>xz</pack-compression-format>
         <uninstaller name="izpack-uninstaller.jar" path="$INSTALL_PATH/lib"/>
     </info>
 

